salt/metalk8s/addons/logging/fluent-bit/deployed/configmap.sls

```diff
@@ -0,0 +1,104 @@
+            app.kubernetes.io/part-of: metalk8s
+            heritage: metalk8s
+            release: fluent-bit
+        data:
+          fluent-bit.conf: |-
+            [SERVICE]
+                HTTP_Server    On
+                HTTP_Listen    0.0.0.0
+                HTTP_PORT      2020
+                Flush          1
+                Daemon         Off
+                Log_Level      warn
+                Parsers_File   parsers.conf
+            [INPUT]
+                Name           tail
+                Tag            kube.*
+                Path           /var/log/containers/*.log
+                Parser         docker
+                DB             /run/fluent-bit/flb_kube.db
+                Mem_Buf_Limit  5MB
+            [INPUT]
+                Name              systemd
+                Tag               host.*
+                DB                /run/fluent-bit/flb_journal.db
+                Strip_Underscores On
+            [FILTER]
+                Name           kubernetes
+                Match          kube.*
+                Kube_URL       https://kubernetes.default.svc:443
+                Merge_Log On
+                K8S-Logging.Parser Off
+            [FILTER]
+                Name           modify
+                Match          host.*
+                Rename         HOSTNAME node
+                Rename         SYSTEMD_UNIT unit
+                Rename         MESSAGE message
+                Remove         CAP_EFFECTIVE
+                Remove         SYSTEMD_SLICE
+                Remove         PID
+                Remove         STREAM_ID
+                Remove         PRIORITY
+                Remove         COMM
+                Remove         CMDLINE
+                Remove         EXE
+                Remove         SYSTEMD_CGROUP
+                Remove         SELINUX_CONTEXT
+                Remove         MACHINE_ID
+                Remove         SYSLOG_IDENTIFIER
+                Remove         TRANSPORT
+                Remove         SYSLOG_FACILITY
+                Remove         BOOT_ID
+                Remove         UID
+                Remove         GID
+            [Output]
+                Name loki
+                Match *
+                Url http://loki:3100/api/prom/push
+                TenantID ""
+                BatchWait 1
+                BatchSize 10240
+                Labels {job="fluent-bit"}
+                RemoveKeys kubernetes,stream,node,unit
+                AutoKubernetesLabels false
+                LabelMapPath /fluent-bit/etc/labelmap.json
+                LineFormat json
+                LogLevel warn
+          labelmap.json: |-
+            {
+              "kubernetes": {
+                "container_name": "container",
+                "host": "node",
+                "labels": {
+                  "app": "app",
+                  "release": "release"
+                },
+                "namespace_name": "namespace",
+                "pod_name": "instance"
```

Comment:
It's a bit weird to have `instance` label being the pod name, why not name it `pod` ?

Reply:
This was defined this way in the ConfigMap generated with the helm chart, I've changed the configuration as little as possible.
But you're right, it's clearer with `pod`, I'll change it.